web/dashboard/src/components/Search.vue

```diff
@@ -0,0 +1,97 @@
+<template>
+  <div class="hello">
+    <h2>Search package behaviour</h2>
+    <b-form-group label="Type">
+      <b-form-radio-group v-model="type">
+        <b-form-radio value="file">File</b-form-radio>
+        <b-form-radio value="command">Command</b-form-radio>
+        <b-form-radio value="ip">IP address</b-form-radio>
+      </b-form-radio-group>
+    </b-form-group>
+    <b-form-input
+      v-model="search"
+      placeholder="Path component (e.g. '.ssh')"
+    ></b-form-input>
+    <b-table :fields="fields" :items="results">
+      <template #cell(ecosystemPkg)="data">
+        {{ data.item.Package.Ecosystem }}/{{ data.item.Package.Name }}
+      </template>
+      <template #cell(version)="data">
+        {{ data.item.Package.Version }}
+        <router-link :to="getPackageLink(data.item.Package)">
+          <b-icon icon="link"></b-icon>
+        </router-link>
+      </template>
+    </b-table>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'Search',
+  props: {},
+  methods: {
+    async makeRequest() {
+      const curId = this.requestId;
+      const backend = process.env.VUE_APP_BACKEND;
+      const response = await fetch(
+          `${backend}/query`, {
```

Comment:
do we need POST for a query, i.e.e read-only request? Can we use GET?

Reply:
POST is also commonly used for query-like operations (e.g. https://cloud.google.com/bigquery/docs/reference/rest/v2/jobs/query).

Also query parameters might become more complex over time, and it'll become unwieldy to keep using GET.